code/modules/library/soapstone.dm

```diff
@@ -0,0 +1,127 @@
+/obj/item/soapstone
+    name = "soapstone"
+    desc = "Leave informative messages for the crew, including the crew of future shifts!\n(Disclaimer: Does not work on shuttles)\n WARNING: Side effects may include beatings, bannings and orbital bombardment."

```

Comment:
Explicitly referring to bannings here feels immersion-breaking, to me. Likewise for the disclaimer in brackets.